[TW-1188] Break up "Private API Network"

akinard-postman opened the pull request:
This PR reorganizes the **Your Private API Network** document by splitting it up into separate pages:

![Screenshot 2023-07-07 at 1 45 23 PM](https://github.com/postmanlabs/postman-docs/assets/113702138/4e4c9797-0b2e-4502-ac71-59d9edfa723e)

For the most part I just took the existing content and moved it to the new articles.

The **Organizing your Private API Network** article is the bulkiest of these documents, but I didn't think that the information contained needed further splitting. I tried to keep the articles contained as follows:

- **Organizing your Private API Network** — Adding, editing, and managing folders and elements.
- **Managing your Private API Network** — Managing user permissions, reporting.
- **Private API Network requests** — Requests (making and managing) to the Private API Network.

For the naming of these articles, I borrowed a little from the **Sharing to the Public API Network** section:

<img width="308" alt="image" src="https://github.com/postmanlabs/postman-docs/assets/113702138/672e37af-7d36-4e5b-b884-1f14af9ed96f">

## Notable updates

While reviewing the docs, I noticed and fixed some things:

- In the **Organizing your Private API Network** article, the links in the “Adding an API/collection/workspace from the workbench” were referencing the wrong info (they don't work in the live article because they all point at `/adding-private-network/#requesting-to-add-elements-in-your-private-api-network`).
- The "Searching, filtering, and sorting" section was a little confusing in its previous state, so I rearranged and revised some of the information (and rearranged images) presented in this section (here's [the link](https://learning.postman.com/docs/collaborating-in-postman/adding-private-network/#searching-filtering-and-sorting) to the live article for context).
- I moved some of the callout info inline on some steps — for example, in the "Organizing with folders" section in the **Organizing your Private API Network** article (for context, [here's the live article](https://learning.postman.com/docs/collaborating-in-postman/adding-private-network/#organizing-with-folders)). It felt a little cluttered up and that info that works just as easily inline with the other instructions.

MatthewPinsker-postman commented on src/pages/docs/collaborating-in-postman/private-api-network/adding-private-network.md:
```diff
@@ -0,0 +1,114 @@
+
+There are several ways to search, filter, and sort elements and folders in the Private API Network. You can search and sort all elements and folders from the left sidebar. You can also select a folder to search, filter, and sort elements on the right.
+
+You can search and sort elements and folders from the left sidebar.
+
+<img alt="Create new folder in Private Network" src="https://assets.postman.com/postman-docs/v10/private-api-network-organize-folders-v10-2.jpg" width="300px"/>
+
+Select <img alt="Home icon" src="https://assets.postman.com/postman-docs/v10/icon-home-v10.jpg#icon" width="16px"> **Home** in the left sidebar to search, filter, and sort all elements in your Private API Network. Select a folder in the left sidebar to search, filter, and sort elements only in the selected folder.
+
+* To search by name, enter the full or partial element or folder name into **Search elements and folders**.
+* To sort elements and folders, select the sort icon <img alt="Sort icon" src="https://assets.postman.com/postman-docs/icon-sort.jpg#icon" width="16px">. Options are:
+    * **Default** - Sort by type, grouping folders and elements by type.
+    * **Date added** - Sort by the date that elements and folders were added.
+    * **A to Z** - Sort elements and folders alphabetically.
```
This section is meant to explain how to use the search bar and sort button (to the right of it) in the left sidebar. The search bar and sort button sort the folders and elements that appear in the left sidebar.

We can definitely rework this section, but I think the paragraph on line 70 isn't related to the bullet point options on lines 72-76.

akinard-postman replied:
Thanks for catching that; I think that was a copy-paste mistake.

MatthewPinsker-postman commented on src/pages/docs/collaborating-in-postman/private-api-network/adding-private-network.md:
```diff
@@ -0,0 +1,114 @@
+    * **Date added** - Sort by the date that elements and folders were added.
+    * **A to Z** - Sort elements and folders alphabetically.
+
+Select a folder from the left sidebar to search, filter, and sort its elements in the folder overview on the right.
+
+<img alt="Private API List" src="https://assets.postman.com/postman-docs/v10/private-api-network-list-v10-2.jpg"/>
+
+* To search by name, enter the full or partial element name into **Search elements**.
+* To filter based on the person who added the element, select **Added by**, then select the name of the team member.
+* To filter by element type, select **Type**, then select **API**, **Collection**, or **Workspace**.
+* To filter based on element tags, select **Tags**, then select the tags. Learn more about adding tags to [collections](/docs/collections/using-collections/#tagging-a-collection), [APIs](/docs/designing-and-developing-your-api/managing-apis/#tagging-apis), and [workspaces](/docs/collaborating-in-postman/using-workspaces/managing-workspaces/#tagging-a-workspace).
+* To sort elements, select the **Sort by** dropdown list. Options are:
+    * **Recently added** - Sort elements by the date they were added.
+    * **A to Z** - Sort elements alphabetically.
+    * **Trending this week** - Sort elements by your team's usage this week.
+    * **Trending this month** - Sort elements by your team's usage this month.
```
This section is meant to explain how to search and filter elements using the options in the bullet points. Users can select either Home or a folder (in the left sidebar) to search /filter elements on the right.

Users can select Home to search and filter all elements in the network. Users can also select a folder to search and filter elements in that folder.

We can definitely rework this, but I think we should also add the detail about selecting Home.

akinard-postman replied:
What about reworking it so it's broken up into a h4 sub-section like this:
![image](https://github.com/postmanlabs/postman-docs/assets/113702138/6ab8223c-a442-4af8-9e65-7288c1e9666b)

MatthewPinsker-postman replied:
Yeah, I like the idea of giving these sections headers!

Also, I'm wondering if we should bring the current H3 headers up to H2 headers. Now that the content has its own page, I'm wondering if we need the same amount of levels.

akinard-postman replied:
That's a good point; I'll update the headers!

akinard-postman commented:
@MatthewPinsker-postman I think this is ready for another looking-over at this point. I moved sections around per your feedback in Slack (https://github.com/postmanlabs/postman-docs/pull/4910/commits/84268e263854b8ae8eb50fa4597de9062d2c051d) and also updated links and the redirect.json file (https://github.com/postmanlabs/postman-docs/pull/4910/commits/8735127015d290b7af624a9eb5929a0f46adf4c8).

MatthewPinsker-postman commented on src/pages/docs/collaborating-in-postman/private-api-network/adding-private-network.md:
```diff
@@ -0,0 +1,130 @@
+
+The _Private API Network_ provides a [central directory](https://www.postman.com/api-platform/api-catalog/) of workspaces, collections, and APIs your team uses internally. Your Postman team can access these resources and start using them right away. By using the Private API Network, you can enable developers across your organization to discover, consume, and track API development in one place.
+
+Workspaces, collections, and APIs in the Private API Network are visible to logged-in users who are on your Postman team. Users who aren't on your team can't find or access these resources. [Super Admins](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) and [API Network Managers](/docs/collaborating-in-postman/roles-and-permissions/#network-roles) can manage all folders and eleme ...
```
Since this page is an overview of the section, what do you think about also mentioning Folder Managers?

akinard-postman replied:
I think that's a good idea. I'll update it.

MatthewPinsker-postman commented on src/pages/docs/collaborating-in-postman/private-api-network/adding-private-network.md:
```diff
@@ -0,0 +1,130 @@
+* Select **Home** from the Postman header, then select **Private API Network** in your team information on the left sidebar.
+* Select **API Network** from the Postman header, then select **Private API Network**.
+
+## Searching, filtering, and sorting
+
+There are several ways to search, filter, and sort elements and folders in the Private API Network. You can search and sort all elements and folders from the left sidebar. You can also select a folder to search, filter, and sort elements on the right.
+
+You can search and sort elements and folders from the left sidebar.
+
+<img alt="Create new folder in Private Network" src="https://assets.postman.com/postman-docs/v10/private-api-network-organize-folders-v10-2.jpg" width="300px"/>
+
+Select <img alt="Home icon" src="https://assets.postman.com/postman-docs/v10/icon-home-v10.jpg#icon" width="16px"> **Home** in the left sidebar to search, filter, and sort all elements in your Private API Network.
+
+* To search by name, enter the full or partial element or folder name into **Search elements and folders**.
+* To sort elements and folders, select the sort icon <img alt="Sort icon" src="https://assets.postman.com/postman-docs/icon-sort.jpg#icon" width="16px">. Options are:
+    * **Default** - Sort by type, grouping folders and elements by type.
+    * **Date added** - Sort by the date that elements and folders were added.
+    * **A to Z** - Sort elements and folders alphabetically.
+
+### Search using the Private API Network overview
+
+You can search, filter, and sort Private API Network folders as well as elements with the overview feature.
+
+* Select <img alt="Home icon" src="https://assets.postman.com/postman-docs/v10/icon-home-v10.jpg#icon" width="16px"> **Home** in the left sidebar to search, filter, and sort all elements in your Private API Network.
+* Select a folder from the left sidebar to search, filter, and sort its elements in the folder overview on the right.
+
+<img alt="Private API List" src="https://assets.postman.com/postman-docs/v10/private-api-network-list-v10-2.jpg"/>
+
+The **All APIs, collections and workspaces** section in the overview lets you search, sort, and filter with the following options:
+
+* To search by name, enter the full or partial element name into **Search elements**.
+* To filter based on the person who added the element, select **Added by**, then select the name of the team member.
+* To filter by element type, select **Type**, then select **API**, **Collection**, or **Workspace**.
+* To filter based on element tags, select **Tags**, then select the tags. Learn more about adding tags to [collections](/docs/collections/using-collections/#tagging-a-collection), [APIs](/docs/designing-and-developing-your-api/managing-apis/#tagging-apis), and [workspaces](/docs/collaborating-in-postman/using-workspaces/managing-workspaces/#tagging-a-workspace).
+* To sort elements, select the **Sort by** dropdown list. Options are:
+    * **Recently added** - Sort elements by the date they were added.
+    * **A to Z** - Sort elements alphabetically.
+    * **Trending this week** - Sort elements by your team's usage this week.
+    * **Trending this month** - Sort elements by your team's usage this month.
```
```suggestion
## Searching, filtering, and sorting

There are several ways to search, filter, and sort elements and folders in the Private API Network. You can search and sort all elements and folders from the left sidebar. You can also select <img alt="Home icon" src="https://assets.postman.com/postman-docs/v10/icon-home-v10.jpg#icon" width="16px"> **Home** or a folder to search, filter, and sort elements on the right.

### Searching for folders and elements in the network

You can search and sort elements and folders in the network from the left sidebar. Then you can select a folder or element to view its details on the right.

<img alt="Create new folder in Private Network" src="https://assets.postman.com/postman-docs/v10/private-api-network-organize-folders-v10-2.jpg" width="300px"/>

* To search by name, select **Search elements and folders** and enter the full or partial element or folder name.
* To sort elements and folders, select the sort icon <img alt="Sort icon" src="https://assets.postman.com/postman-docs/icon-sort.jpg#icon" width="16px">. Options are:
    * **Default** - Sort by type, grouping folders and elements by type.
    * **Date added** - Sort by the date that elements and folders were added.
    * **A to Z** - Sort elements and folders alphabetically.

### Searching for elements in a folder

You can select a folder to search, filter, and sort elements on the right. Select <img alt="Home icon" src="https://assets.postman.com/postman-docs/v10/icon-home-v10.jpg#icon" width="16px"> **Home** in the left sidebar to search, filter, and sort all elements in your Private API Network. Select a folder in the left sidebar to search, filter, and sort elements only in the selected folder.

You can select a folder in the left sidebar to then search, filter, and sort elements on the right.

* Select <img alt="Home icon" src="https://assets.postman.com/postman-docs/v10/icon-home-v10.jpg#icon" width="16px"> **Home** in the left sidebar to search, filter, and sort all elements in your Private API Network on the right.
* Select a folder from the left sidebar to search, filter, and sort its elements on the right.

<img alt="Private API List" src="https://assets.postman.com/postman-docs/v10/private-api-network-list-v10-2.jpg"/>

The **All APIs, collections and workspaces** section enables you to search, sort, and filter elements with the following options:

* To search by name, enter the full or partial element name into **Search elements**.
* To filter based on the person who added the element, select **Added by**, then select the name of the team member.
* To filter by element type, select **Type**, then select **API**, **Collection**, or **Workspace**.
* To filter based on element tags, select **Tags**, then select the tags. Learn more about adding tags to [collections](/docs/collections/using-collections/#tagging-a-collection), [APIs](/docs/designing-and-developing-your-api/managing-apis/#tagging-apis), and [workspaces](/docs/collaborating-in-postman/using-workspaces/managing-workspaces/#tagging-a-workspace).
* To sort elements, select the **Sort by** dropdown list. Options are:
    * **Recently added** - Sort elements by the date they were added.
    * **A to Z** - Sort elements alphabetically.
    * **Trending this week** - Sort elements by your team's usage this week.
    * **Trending this month** - Sort elements by your team's usage this month.
```

MatthewPinsker-postman replied:
@akinard-postman I think a few details aren't exactly right in the previous version. What do you think about these suggestions?

akinard-postman replied:
I like this suggestion, thank you!

MatthewPinsker-postman commented on src/pages/docs/collaborating-in-postman/private-api-network/adding-private-network.md:
```diff
@@ -0,0 +1,132 @@
+
+## Contents
+
+* [Navigating the Private API Network](#navigating-the-private-api-network)
+* [Searching, filtering, and sorting](#searching-filtering-and-sorting)
+* [Reviewing details about elements](#reviewing-details-about-elements)
+* [Sharing folders and elements](#sharing-folders-and-elements)
```
```suggestion
* [Navigating the Private API Network](#navigating-the-private-api-network)
* [Searching, filtering, and sorting](#searching-filtering-and-sorting)
    * [Searching for folders and elements in the network](#searching-for-folders-and-elements-in-the-network)
    * [Searching for elements in a folder](#searching-for-elements-in-a-folder)
* [Reviewing details about elements](#reviewing-details-about-elements)
* [Sharing folders and elements](#sharing-folders-and-elements)
```

MatthewPinsker-postman commented on src/pages/docs/collaborating-in-postman/private-api-network/organizing-private-network.md:
```diff
@@ -0,0 +1,218 @@
+
+> **[The Private API Network is available on Postman Enterprise plans.](https://www.postman.com/pricing)**
+
+Organizing your Private API Network with folders and elements helps your team easily discover and use your internal workspaces, collections, and APIs.
```
```suggestion
Add elements to the Private API Network to help your team discover and use your team's workspaces, collections, and APIs. You can organize related elements into folders.

[Super Admins](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) and [API Network Managers](/docs/collaborating-in-postman/roles-and-permissions/#network-roles) can manage all folders and elements in your Private API Network. You can assign team members the [Folder Manager role](/docs/collaborating-in-postman/roles-and-permissions/#network-roles) at the folder level, giving them permission to manage specific folders and the elements in them.
```

MatthewPinsker-postman replied:
I'm suggesting this because we usually we add mention about the roles you'll need for the tasks on the page. Thanks!

MatthewPinsker-postman commented on src/pages/docs/collaborating-in-postman/private-api-network/organizing-private-network.md:
```diff
@@ -0,0 +1,218 @@
+
+## Editing Folder Managers
+
+> You can't add Folder Managers to sub-folders.
+
+API Network Managers can assign team members the [Folder Manager role](/docs/collaborating-in-postman/roles-and-permissions/#network-roles), giving them permission to manage specific folders and the elements in them. Folder Managers can also assign other team members the Folder Manager role in folders they have permission to manage.
```
```suggestion
API Network Managers can assign team members the [Folder Manager role](/docs/collaborating-in-postman/roles-and-permissions/#network-roles), giving them permission to manage specific folders and the elements in them. Folder Managers can also assign other team members the Folder Manager role in folders they have permission to manage.

> You can't add Folder Managers to sub-folders.
```

MatthewPinsker-postman replied:
I'm suggesting this because we usually don't begin sections with notes, unless the notes discuss plan availability.

akinard-postman replied:
Got it, thanks for clarification!

MatthewPinsker-postman commented on src/pages/docs/collaborating-in-postman/private-api-network/organizing-private-network.md:
```diff
@@ -0,0 +1,218 @@
+
+1. Select the collections, APIs, and workspaces you want to add. You can search for elements, and filter elements by tags. Learn more about adding tags to [collections](/docs/collections/using-collections/#tagging-a-collection), [APIs](/docs/designing-and-developing-your-api/managing-apis/#tagging-apis), and [workspaces](/docs/collaborating-in-postman/using-workspaces/managing-workspaces/#tagging ...
+
+    If you're adding collections, you can optionally select environments in their workspace to make sure users have access to environment variables. You can also [edit a collection's environments later](#editing-collections).
```
```suggestion
    If you're adding collections, you can optionally select environments in their workspace to make sure users have access to environment variables. You can also [edit a collection's environments later](/docs/collaborating-in-postman/private-api-network/managing-private-network/#editing-collections).
```

akinard-postman replied:
Thanks for catching that. 👍

MatthewPinsker-postman commented on src/pages/docs/collaborating-in-postman/private-api-network/organizing-private-network.md:
```diff
@@ -0,0 +1,218 @@
+    <img alt="Add collection to the Private Network" src="https://assets.postman.com/postman-docs/v10/add-collection-to-private-api-network-v10.jpg"/>
+
+1. (Optional) Add a brief summary about the collection.
+1. (Optional) Select **Select Environments** to make sure users have access to environment variables. You can also edit a collection's summary and environments later. To learn more, see [Editing collections](#editing-collections).
```
```suggestion
1. (Optional) Select **Select Environments** to make sure users have access to environment variables. You can also edit a collection's summary and environments later. To learn more, see [Editing collections](/docs/collaborating-in-postman/private-api-network/managing-private-network/#editing-collections).
```

MatthewPinsker-postman commented on src/pages/docs/collaborating-in-postman/private-api-network/managing-private-network.md:
```diff
@@ -0,0 +1,83 @@
+
+> **[The Private API Network is available on Postman Enterprise plans.](https://www.postman.com/pricing)**
+
+[Super Admins](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) and [API Network Managers](/docs/collaborating-in-postman/roles-and-permissions/#network-roles) can manage all folders and elements in your Private API Network. You can assign team members the [Folder Manager role](/docs/collaborating-in-postman/roles-and-permissions/#network-roles) at the folder level, giving them p ...
```
```suggestion
[Super Admins](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) and [API Network Managers](/docs/collaborating-in-postman/roles-and-permissions/#network-roles) can manage all folders and elements in your Private API Network. You can assign team members the [Folder Manager role](/docs/collaborating-in-postman/roles-and-permissions/#network-roles) at the folder level, giving them permission to manage specific folders and the elements in them. Users with the Editor role for an element can also manage the element.
```

MatthewPinsker-postman commented on src/pages/docs/collaborating-in-postman/private-api-network/managing-private-network.md:
```diff
@@ -0,0 +1,83 @@
+
+[Super Admins](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) and [API Network Managers](/docs/collaborating-in-postman/roles-and-permissions/#network-roles) can manage all folders and elements in your Private API Network. You can assign team members the [Folder Manager role](/docs/collaborating-in-postman/roles-and-permissions/#network-roles) at the folder level, giving them p ...
+
+> Enterprise teams can also assign this role to a user group. For more information about assigning team roles to groups, see [User groups](/docs/collaborating-in-postman/user-groups/).
```
```suggestion
> Enterprise teams can also assign the API Network Manager role to a user group. For more information about assigning team roles to groups, see [User groups](/docs/collaborating-in-postman/user-groups/).
```

MatthewPinsker-postman commented on src/pages/docs/collaborating-in-postman/private-api-network/managing-private-network.md:
```diff
@@ -0,0 +1,83 @@
+
+---
+
+> **[The Private API Network is available on Postman Enterprise plans.](https://www.postman.com/pricing)**
+
```
```suggestion
> **[The Private API Network is available on Postman Enterprise plans.](https://www.postman.com/pricing)**

You can manage folders and elements in the Private API Network. You can edit a collection's summary and environments, move folders and elements, rename folders, and remove folders and elements from the network.
```

MatthewPinsker-postman replied:
What do you think about adding a brief intro to the page?

akinard-postman replied:
I agree; I wasn't too completely sure about the repurposed statement from the original page format (beginning with "Super Admins and API Network Managers...").

MatthewPinsker-postman commented on src/pages/docs/collaborating-in-postman/private-api-network/managing-private-network.md:
```diff
@@ -0,0 +1,83 @@
+
+> Enterprise teams can also assign this role to a user group. For more information about assigning team roles to groups, see [User groups](/docs/collaborating-in-postman/user-groups/).
+
+## Contents
+
+* [Managing folders and elements](#managing-folders-and-elements)
+    * [Editing collections](#editing-collections)
+    * [Moving folders and elements](#moving-folders-and-elements)
+    * [Renaming folders](#renaming-folders)
+    * [Removing folders and elements](#removing-folders-and-elements)
+
+## Managing folders and elements
+
+API Network Managers can manage all folders and elements. Folder Managers can manage folders and elements in folders they have permission to manage. Users with the Editor role for an element can also manage the element.
+
+> You can manage elements within the [Private API Network](/docs/collaborating-in-postman/private-api-network/adding-private-network/#navigating-the-private-api-network) or with the [Postman API](https://www.postman.com/postman/workspace/postman-public-workspace/folder/12959542-b7c02959-88ca-4e2f-9b68-99538eed4533?ctx=documentation).
```
What do you think about moving the note to the bottom of the intro section, and then changing the H3 headers to H2? (And then moving them up a level in the Contents section)

akinard-postman replied:
I agree! I wasn't too sure about the current format for the TOC, but wasn't sure as well whether I should make them all H2. I'll make that update.

MatthewPinsker-postman commented on src/pages/docs/collaborating-in-postman/private-api-network/adding-private-network.md:
```diff
@@ -0,0 +1,134 @@
+
+The _Private API Network_ provides a [central directory](https://www.postman.com/api-platform/api-catalog/) of workspaces, collections, and APIs your team uses internally. Your Postman team can access these resources and start using them right away. By using the Private API Network, you can enable developers across your organization to discover, consume, and track API development in one place.
+
+Workspaces, collections, and APIs in the Private API Network are visible to logged-in users who are on your Postman team. Users who aren't on your team can't find or access these resources. [Super Admins](/docs/collaborating-in-postman/roles-and-permissions/#team-roles), [API Network Managers](/docs/collaborating-in-postman/roles-and-permissions/#network-roles), and [Folder Managers](/docs/collab ...
```
```suggestion
Workspaces, collections, and APIs in the Private API Network are visible to logged-in users who are on your Postman team. Users who aren't on your team can't find or access these resources.

[Super Admins](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) and [API Network Managers](/docs/collaborating-in-postman/roles-and-permissions/#network-roles) can manage all folders and elements in your Private API Network. You can assign team members the [Folder Manager role](/docs/collaborating-in-postman/roles-and-permissions/#network-roles) at the folder level, giving them permission to manage specific folders and the elements in them.
```

MatthewPinsker-postman commented on src/pages/docs/collaborating-in-postman/private-api-network/private-network-requests.md:
```diff
@@ -0,0 +1,131 @@
+---
+title: "Private API Network requests"
```
What do you think about changing the page title to something like: "Requesting to add to the Private API Network" ?

MatthewPinsker-postman commented on src/pages/docs/collaborating-in-postman/private-api-network/private-network-requests.md:
```diff
@@ -0,0 +1,131 @@
+
+---
+
+> **[The Private API Network is available on Postman Enterprise plans.](https://www.postman.com/pricing)**
+
```
```suggestion
> **[The Private API Network is available on Postman Enterprise plans.](https://www.postman.com/pricing)**

You can request to add elements to the Private API Network. An API Network Manager or Folder Manager will review your request and either approve or deny it.

```

MatthewPinsker-postman replied:
What do you think about adding a brief intro to the page?

Note: I'm not sure if I got the spacing correct in my suggestion. My intention is to have a space between the suggested line and the line about the Editor role.

akinard-postman replied:
I updated this in 02dcb995cad3351936b8bc4bbabb110e749b8438; let me know if that looks OK? I wasn't 100% sure whether you intended to remove the statement about Editors or leave it in place (since it was mentioned in a [previous comment](https://github.com/postmanlabs/postman-docs/pull/4910#discussion_r1260219393)). Thank you!

MatthewPinsker-postman replied:
Sorry for the confusion. I was suggesting the following:

```
> **[The Private API Network is available on Postman Enterprise plans.](https://www.postman.com/pricing)**

You can request to add elements to the Private API Network. An API Network Manager or Folder Manager will review your request and either approve or deny it.

Users with the [Editor role](/docs/collaborating-in-postman/roles-and-permissions/#workspace-roles) for an element can request to add elements to the Private API Network.
```

MatthewPinsker-postman commented on src/pages/docs/collaborating-in-postman/private-api-network/private-network-requests.md:
```diff
@@ -0,0 +1,131 @@
+
+1. Search for and then select collections, APIs, and workspaces you want to request to add. You can also filter elements by tags. Learn more about adding tags to [collections](/docs/collections/using-collections/#tagging-a-collection), [APIs](/docs/designing-and-developing-your-api/managing-apis/#tagging-apis), and [workspaces](/docs/collaborating-in-postman/using-workspaces/managing-workspaces/# ...
+
+    If you're requesting to add collections, you can optionally select environments in their workspace to make sure users have access to environment variables. You can also [edit a collection's environments later](#editing-collections).
```
```suggestion
    If you're requesting to add collections, you can optionally select environments in their workspace to make sure users have access to environment variables. You can also [edit a collection's environments later](/docs/collaborating-in-postman/private-api-network/managing-private-network/#editing-collections).
```

MatthewPinsker-postman commented on src/pages/docs/collaborating-in-postman/private-api-network/private-network-requests.md:
```diff
@@ -0,0 +1,131 @@
+1. (Optional) Add a brief summary about the collection.
+1. (Optional) Select **Select Environments** to make sure users have access to environment variables.
+
+    > You can also edit a collection's summary and environments later. To learn more, see [Editing collections](#editing-collections).
```
```suggestion
    > You can also edit a collection's summary and environments later. To learn more, see [Editing collections](/docs/collaborating-in-postman/private-api-network/managing-private-network/#editing-collections).
```

akinard-postman commented on src/pages/docs/collaborating-in-postman/private-api-network/private-network-requests.md:
```diff
@@ -0,0 +1,131 @@
+
+> **[The Private API Network is available on Postman Enterprise plans.](https://www.postman.com/pricing)**
+
+Users with the [Editor role](/docs/collaborating-in-postman/roles-and-permissions/#workspace-roles) for an element can request to add elements to the Private API Network.
```
Removing it here is fine with me since, yeah, there's links in each section for users to follow out to.

MatthewPinsker-postman commented on src/pages/docs/collaborating-in-postman/private-api-network/managing-private-network.md:
```diff
@@ -0,0 +1,84 @@
+
+> **[The Private API Network is available on Postman Enterprise plans.](https://www.postman.com/pricing)**
+
+You can manage folders and elements in the Private API Network. You can edit a collection's summary and environments, move folders and elements, rename folders, and remove folders and elements from the network.
+[Super Admins](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) and [API Network Managers](/docs/collaborating-in-postman/roles-and-permissions/#network-roles) can manage all folders and elements in your Private API Network. You can assign team members the [Folder Manager role](/docs/collaborating-in-postman/roles-and-permissions/#network-roles) at the folder level, giving them p ...
```
```suggestion
You can manage folders and elements in the Private API Network. You can edit a collection's summary and environments, move folders and elements, rename folders, and remove folders and elements from the network.

[Super Admins](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) and [API Network Managers](/docs/collaborating-in-postman/roles-and-permissions/#network-roles) can manage all folders and elements in your Private API Network. You can assign team members the [Folder Manager role](/docs/collaborating-in-postman/roles-and-permissions/#network-roles) at the folder level, giving them permission to manage specific folders and the elements in them. Users with the Editor role for an element can also manage the element.
```

MatthewPinsker-postman commented on src/pages/docs/collaborating-in-postman/private-api-network/managing-private-network.md:
```diff
@@ -0,0 +1,84 @@
+You can manage folders and elements in the Private API Network. You can edit a collection's summary and environments, move folders and elements, rename folders, and remove folders and elements from the network.
+[Super Admins](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) and [API Network Managers](/docs/collaborating-in-postman/roles-and-permissions/#network-roles) can manage all folders and elements in your Private API Network. You can assign team members the [Folder Manager role](/docs/collaborating-in-postman/roles-and-permissions/#network-roles) at the folder level, giving them p ...
+
+> Enterprise teams can also assign the API Network Manager role to a user group. For more information about assigning team roles to groups, see [User groups](/docs/collaborating-in-postman/user-groups/).
+
+## Contents
+
+* [Managing folders and elements](#managing-folders-and-elements)
+* [Editing collections](#editing-collections)
+* [Moving folders and elements](#moving-folders-and-elements)
+* [Renaming folders](#renaming-folders)
+* [Removing folders and elements](#removing-folders-and-elements)
+
+## Managing folders and elements
+
+API Network Managers can manage all folders and elements. Folder Managers can manage folders and elements in folders they have permission to manage. Users with the Editor role for an element can also manage the element.
+
+> You can manage elements within the [Private API Network](/docs/collaborating-in-postman/private-api-network/adding-private-network/#navigating-the-private-api-network) or with the [Postman API](https://www.postman.com/postman/workspace/postman-public-workspace/folder/12959542-b7c02959-88ca-4e2f-9b68-99538eed4533?ctx=documentation).
```
```suggestion
> Enterprise teams can also assign the API Network Manager role to a user group. For more information about assigning team roles to groups, see [User groups](/docs/collaborating-in-postman/user-groups/).

<!-- -->

> You can manage elements within the [Private API Network](/docs/collaborating-in-postman/private-api-network/adding-private-network/#navigating-the-private-api-network) or with the [Postman API](https://www.postman.com/postman/workspace/postman-public-workspace/folder/12959542-b7c02959-88ca-4e2f-9b68-99538eed4533?ctx=documentation).

## Contents

* [Editing collections](#editing-collections)
* [Moving folders and elements](#moving-folders-and-elements)
* [Renaming folders](#renaming-folders)
* [Removing folders and elements](#removing-folders-and-elements)
```

MatthewPinsker-postman replied:
What do you think about removing this section since the same information is also in the intro?

akinard-postman replied:
Good call!

MatthewPinsker-postman commented on src/components/LeftNav/LeftNavItems.jsx:
```diff
@@ -684,7 +684,26 @@ export const leftNavItems = [
+            url: '/docs/collaborating-in-postman/private-api-network/managing-private-network/'
+          },
+          {
+            name: 'Private API Network requests',
```
```suggestion
            name: 'Requesting to add to the Private API Network',
```

akinard-postman replied:
Thanks for catching that!

MatthewPinsker-postman commented on src/pages/docs/collaborating-in-postman/private-api-network/private-network-requests.md:
```diff
@@ -0,0 +1,131 @@
+
+### Requesting to add a workspace from the workbench
+
+[Workspace Editors](/docs/collaborating-in-postman/roles-and-permissions/#workspace-roles) can request to add workspaces to your Private API Network from the workbench.
```
```suggestion
[Workspace Editors](/docs/collaborating-in-postman/roles-and-permissions/#workspace-roles) can request to add workspaces to your Private API Network from the workbench. API Network Managers and Folder Managers can [add a workspace](/docs/collaborating-in-postman/private-api-network/organizing-private-network/#adding-a-workspace-from-the-workbench).
```

MatthewPinsker-postman replied:
I'm suggesting this because there are a few section on collections, APIs, and workspace pages where we link to the request instructions. I think this makes sense because most users will only have the Editor role for an element.

In case API Network Managesr or Folder Managers navigate to the requests-specific sections, I'm wondering if we should help point them to instructions about directly adding elements to the network.

What do you think?

akinard-postman replied:
I agree with that assessment.

MatthewPinsker-postman commented on src/pages/docs/collaborating-in-postman/private-api-network/private-network-requests.md:
```diff
@@ -0,0 +1,131 @@
+
+### Requesting to add a collection from the workbench
+
+[Collection Editors](/docs/collaborating-in-postman/roles-and-permissions/#collection-roles) can request to add collections to your Private API Network from the workbench.
```
```suggestion
[Collection Editors](/docs/collaborating-in-postman/roles-and-permissions/#collection-roles) can request to add collections to your Private API Network from the workbench. API Network Managers and Folder Managers can [add a collection](/docs/collaborating-in-postman/private-api-network/organizing-private-network/#adding-a-collection-from-the-workbench).
```

MatthewPinsker-postman commented on src/pages/docs/collaborating-in-postman/private-api-network/private-network-requests.md:
```diff
@@ -0,0 +1,131 @@
+
+### Requesting to add an API from the workbench
+
+[API Editors](/docs/collaborating-in-postman/roles-and-permissions/#api-roles) can request to add APIs to your Private API Network from the workbench.
```
```suggestion
[API Editors](/docs/collaborating-in-postman/roles-and-permissions/#api-roles) can request to add APIs to your Private API Network from the workbench. API Network Managers and Folder Managers can [add an API](/docs/collaborating-in-postman/private-api-network/organizing-private-network/#adding-an-api-from-the-workbench).
```

MatthewPinsker-postman commented on redirects.json:
```diff
@@ -503,6 +503,10 @@
     "from": "/docs/postman/collaboration/adding_private_network/",
     "to": "/docs/collaborating-in-postman/adding-private-network/"
```
```suggestion
    "to": "/docs/collaborating-in-postman/private-api-network/adding-private-network/"
```

MatthewPinsker-postman replied:
I think we also need to add the same path to line `500`. Thanks!

MatthewPinsker-postman replied:
Ya I think we want all redirects to point to the latest URL, instead of older redirects bouncing from A -> B -> C.

MatthewPinsker-postman commented on redirects.json:
```diff
@@ -501,7 +501,11 @@
   },
```
@akinard-postman I think we also want to update the path on line 500 to `/docs/collaborating-in-postman/private-api-network/adding-private-network/`

I can't add an inline suggestion on this line. Thanks!

akinard-postman replied:
I've updated this in 6f9f216d6fca21c7a1c59bdca47d281898cedc1c — thanks for catching this!

MatthewPinsker-postman reviewed (approved):
LGTM! Thanks for working on this!

akinard-postman commented:
Per @jkonrath-postman tagging this as HOLD for release after the push on Monday (17/7/2023).
